discovery/sync_manager.go

```diff
@@ -32,12 +32,12 @@ const (
 	// This is the most that can be sent in a given go. Requests beyond
 	// this, will block indefinitely. Once tokens (bytes are depleted),
 	// they'll be refilled at the DefaultMsgBytesPerSecond rate.
-	DefaultMsgBytesBurst = 2 * 100 * 1_024
+	DefaultMsgBytesBurst = 2 * 1000 * 1_024
```

Comment:
Why do we need to also increase this?

Reply:
the burst must be greater than the `msg-rate-bytes` otherwise `msg-rate-bytes` may not take effect, more details [here](https://github.com/lightningnetwork/lnd/pull/10097#discussion_r2222035250).